contributor/es/06-conclusion-script-es.asciidoc

```diff
@@ -0,0 +1,35 @@
+# Transcripción: El contribuidor en InnerSource – Introducción 
+
+=== Duración: 1-3 Minutos
+
+=== Actores: Isabel y Johannes
+
+=== Resumen del “PlayBook”
+
+Este segmento recapitula que hemos aprendido acerca de ser un contribuidor InnerSource. Compartiremos como puedes continuar tu aprendizaje en InnerSource, ambos con otros videos en línea y también el involucrarse con la comunidad en línea de InnerSource.
+
+=== Transcripción
+
+J: Gracias por ver el segmento de contribuidor de la ruta de aprendizaje de InnerSource Commons. Con esta sección aprendiste sobre el rol de contribuidor – La sangre vital de los proyectos de InnerSource:
+Externo a los propietarios de los integrantes del equipo, ellos dan una valiosa entrada al proyecto.
+
+I: Usted ha aprendido a cómo convertirse en un colaborador buscando oportunidades para contribuir, así mismo  como la mentalidad y los hábitos necesarios para encontrar o crear tales oportunidades.
+También discutimos la ética del rol y sugerimos algún comportamiento que probablemente le conduzca a contribuciones exitosas.
+
+J: Dada la mentalidad correcta, los hábitos y la ética aún hay algunos detalles que podrían evitar que contribuya con éxito; por lo tanto, hemos discutido estos aspectos básicos con mayor detalle.
+
+I: Por último, pero no menos importante, el convencer a tus compañeros de equipo y a su organización a varios niveles puede ser difícil, por lo que hemos discutido los beneficios de la contribución desde varias perspectivas para hacerlo más fácil para ti.
```

Comment:
It's ok as it is.
I suggest "hacerlo más fácil para ti." -> "facilitarte el proceso.".
Yet feel free to ignore this one.